### PR TITLE
Merge Dev into Master

### PR DESCRIPTION
**Pull request by haydenudelson**
#### What does this PR do?

Merges DEV into master, with Alex's theming update, event component back-end, fleshed out team wallboards.

#### Who is reviewing it?

@vinamartin @alchucam @mcalcote @haydenudelson

#### How should this be tested?

`master go`

#### Checklist:
- [ ] Documentation Updated

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: client/utils/Link.js [hunk 1]
```diff
@@ -26,13 +26,40 @@ const REFLEXURL =
 export const IONURL =
   "http://jenkins.phx.connexta.com/service/jenkins/blue/rest/organizations/jenkins/pipelines/ION/";
 
+<<<<<<< HEAD
```
Comment (alchucam) [inline client/utils/Link.js]: ✏️ Not cleared

File: client/utils/Link.js [hunk 2]
```diff
@@ -26,13 +26,40 @@ const REFLEXURL =
...
+    NAME: "11.x",
+    URL:
+      "http://jenkins.phx.connexta.com/service/jenkins/blue/rest/organizations/jenkins/pipelines/GSR/pipelines/11.x/"
+  },
+  {
+    NAME: "13.x",
+    URL:
+      "http://jenkins.phx.connexta.com/service/jenkins/blue/rest/organizations/jenkins/pipelines/GSR/pipelines/13.x/"
+  }
+=======
```
Comment (alchucam) [inline client/utils/Link.js]: ✏️ Not cleared

File: client/utils/Link.js [hunk 3]
```diff
@@ -26,13 +26,40 @@ const REFLEXURL =
...
 export const jenkinsURLList = [
   { Alliance: AllianceURL },
   { DDF: DDFURL },
   { DIB: DIBURL },
   { GSR: GSRURL },
   { SOAESB: SOAESBURL },
   { REFLEX: REFLEXURL }
+>>>>>>> 737ec059a306c8b7870a5c39b31c274745d27b9a
```
Comment (alchucam) [inline client/utils/Link.js]: ✏️ Not cleared

**Comment by alchucam** [comment]
❓ I thought we removed resources/carouselMedia images, and left just the cute EVE picture as it doesn't matter for persistent data?

File: resources/carousel.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    },
+
+    {
+      "title": "Fun at the offsite!",
+      "body": "Great views from the company offsite!",
+      "media": "Offsite.jpg",
+      "link": ""
+    }
+  ]
+}
```
Comment (alchucam) [inline resources/carousel.json]: ❓  do we still need these default data?

**Comment by alchucam** [comment]
❗️ GSR wallboard didn't get applied with the newest changes.

**Comment by haydenudelson** [comment]
To address Alex's concerns- I forgot to push after I resolved some of the merge conflicts.  Hopefully it's good now!

**Pull request merged**